Update efs blueprint to use TroposphereType as FileSystem variable type

Evnsan opened the pull request.

Evnsan committed:
Add yaml based test for efs blueprint

Evnsan committed:
Update efs blueprint to use TroposphereType as variable for FileSystem

stoiev commented on stacker_blueprints/efs.py:
```diff
@@ -121,11 +123,12 @@ def create_efs_filesystem(self):
-            PerformanceMode=v['PerformanceMode']))
+        fs = v.get('FileSystem')
+
+        # This is a major hack to inkect extra tags in efs resources
```
```suggestion
        # This is a major hack to inject extra tags in efs resources
```

Evnsan committed:
Add support for passing subnets as comma separated string

Evnsan committed:
Remove duplicant tests for efs

Remove success tests for test_efs.py since we are using yaml test for
success cases
